### PR TITLE
feature/revamp

### PR DESCRIPTION
**Pull request by Spadavecchia**
Changed framework bootstrap-vue -> Vuetify and minor corrections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,129 @@
...
 
-## Installation
+### [Screenshots](https://github.com/spadaveccia/roger_ui/tree/master/screenshots)
+
+## Features
+
+- Graphical User Interface
+- Multi-tentant architecture, based on partitions or nodes
+- Based on Roger and RabbitMQ
+- On-line monitoring - Per-queue concurrency control
```
Comment (Eptis) [inline README.md:15]: `On-line` should be `Online`

File: README.md [hunk 2]
```diff
@@ -1,26 +1,129 @@
...
+- Jobs cancellation (both queued and while running)
```
Comment (Eptis) [inline README.md:16]: `Jobs` should be `Job`

File: docs/screenshots/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+### Main Screen
+![Main-Screen](https://raw.githubusercontent.com/fabianherrera/roger_ui/develop/docs/screenshots/main.png)
```
Comment (Eptis) [inline docs/screenshots/README.md:2]: Shouldn't the screenshots be updated to show the latest UI?

File: lib/roger_ui/jobs.ex [hunk 4]
```diff
@@ -1,6 +1,53 @@
-defmodule RogerUi.Jobs do
+defmodule RogerUI.Jobs do
   @moduledoc """
-  Generate Jobs list from Roger.Info.running_jobs()
+  Generates a Jobs list from Roger.Info.running_jobs() function.
+  Resulting Jobs are linked to a partition name:
+
+  Given a nested data structure, where each element contents nested items:
+  [
```
Comment (Eptis) [inline lib/roger_ui/jobs.ex:7]: Can you put code examples in markdown code blocks?

File: lib/roger_ui/jobs.ex [hunk 5]
```diff
@@ -1,6 +1,53 @@
...
+    queued_at: 1517697586453,
+    retry_count: 0,
+    started_at: 1517697682999
+  }]
+  """
+
+  @doc """
+  Takes a Keyword list with nodes, partitions and jobs like this:
+
+  [
```
Comment (Eptis) [inline lib/roger_ui/jobs.ex:39]: Can you put code examples in markdown code blocks?

File: lib/roger_ui/jobs.ex [hunk 6]
```diff
@@ -1,6 +1,53 @@
...
+        "roger_demo_partition" => [
+          %Roger.Job{
+            args: %{"country" => "Venezuela", "email" => "janedoe@gmail.com", "name" => "Jane Doe", "number_of_pets" => 966},
+            execution_key: nil, id: "16ovjr39jvijf4kgrlqe2ib2aadojhid", module: RogerDemo.Job.CreateUpdateUser,
+            queue_key: nil, queued_at: 1517697586453, retry_count: 0, started_at: 1517697682999
+          }
+  ]]
+
+  return a sorted Map with the following structure:
+  [
```
Comment (Eptis) [inline lib/roger_ui/jobs.ex:18]: Can you put code examples in markdown code blocks?

File: lib/roger_ui/partitions.ex [hunk 7]
```diff
@@ -0,0 +1,40 @@
+defmodule RogerUI.Partitions do
+  @moduledoc """
+  Generate Partition list from Roger.Info.partitions()
+  """
+  @doc """
+  Takes a Keyword list that contains the nodes, status and partitions with queues, like this:
```
Comment (Eptis) [inline lib/roger_ui/partitions.ex:6]: Can you put code examples in markdown code blocks?

File: lib/roger_ui/partitions.ex [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+          other: %{consumer_count: 1, max_workers: 2, message_count: 0, paused: false}
+        }
+      },...
+    ]
+
+  and transforms it into a partition list
+  """
+  def normalize(partitions) do
+    partitions
+    |> Enum.flat_map(&capture_node_name(&1))
```
Comment (Eptis) [inline lib/roger_ui/partitions.ex:23]: `&capture_node_name(&1)` is redundant and can be written as `&capture_node_name/1`

File: lib/roger_ui/queues.ex [hunk 9]
```diff
@@ -1,18 +1,54 @@
-defmodule RogerUi.Queues do
+defmodule RogerUI.Queues do
   @moduledoc """
-  Normalize queues from nodes
+  Normalizes nodes data structures from Roger.Info.running_jobs() function in order to obtain queues:
+
+  Given a nested data structure, where each element contents nested items:
+    input = [
```
Comment (Eptis) [inline lib/roger_ui/queues.ex:6]: Can you put code examples in markdown code blocks?

File: lib/roger_ui/queues.ex [hunk 10]
```diff
@@ -1,18 +1,54 @@
...
+      "server@127.0.0.1": %{
+        running: %{
+          "roger_test_partition_1" => %{
+            default: %{consumer_count: 1, max_workers: 10, message_count: 740, paused: false},
+            fast: %{consumer_count: 1, max_workers: 10, message_count: 740, paused: false},
+            other: %{consumer_count: 1, max_workers: 2, message_count: 0, paused: false}
+          }]
+
+  return a sorted Map with the following keys:
+  [
```
Comment (Eptis) [inline lib/roger_ui/queues.ex:16]: Can you put code examples in markdown code blocks?

File: lib/roger_ui/queues.ex [hunk 11]
```diff
@@ -1,18 +1,54 @@
...
+  %{"partition_name" => "roger_partition_demo", "queue_name" => "roger_test_partition_1", "qualified_queue_name" => "roger_test_partition_1-default"}
+  }]
   """
+
   alias Roger.Queue
 
+  @doc """
+  Takes a Keyword list that contains the nodes, status and partitions with queues, like this:
+
+   [
```
Comment (Eptis) [inline lib/roger_ui/queues.ex:26]: Can you put code examples in markdown code blocks?

File: priv/vue/src/App.vue [hunk 12]
```diff
@@ -1,40 +1,60 @@
...
+    </v-content>
+  </v-app>
 </template>
 
 <script>
-import ModalError from '@/components/ModalError'
-import loading from 'vue-full-loading'
-import { mapGetters } from 'vuex'
+import ErrorMessage from '@/components/ErrorMessage'
+// import loading from 'vue-full-loading'
```
Comment (Eptis) [inline priv/vue/src/App.vue:43]: Commented line can be removed.

**Review by Eptis — CHANGES_REQUESTED** [review]

**Comment by Eptis** [comment]
Also could you add a compiled version of the frontend so it can be consumed by any elixir project without first having to compile it?

File: README.md (hunk 1 above)
Comment (fabianherrera) [reply README.md:15]: done

File: README.md (hunk 2 above)
Comment (fabianherrera) [reply README.md:16]: done

File: docs/screenshots/README.md (hunk 3 above)
Comment (fabianherrera) [reply docs/screenshots/README.md:2]: done. All screenshots and their links to bettyblock organization were included.

File: lib/roger_ui/jobs.ex (hunk 4 above)
Comment (fabianherrera) [reply lib/roger_ui/jobs.ex:7]: done

File: lib/roger_ui/jobs.ex (hunk 5 above)
Comment (fabianherrera) [reply lib/roger_ui/jobs.ex:39]: done

File: lib/roger_ui/partitions.ex (hunk 8 above)
Comment (fabianherrera) [reply lib/roger_ui/partitions.ex:23]: done

File: lib/roger_ui/queues.ex (hunk 9 above)
Comment (fabianherrera) [reply lib/roger_ui/queues.ex:6]: done

File: lib/roger_ui/queues.ex (hunk 10 above)
Comment (fabianherrera) [reply lib/roger_ui/queues.ex:16]: done

File: lib/roger_ui/queues.ex (hunk 11 above)
Comment (fabianherrera) [reply lib/roger_ui/queues.ex:26]: done

File: priv/vue/src/App.vue (hunk 12 above)
Comment (fabianherrera) [reply priv/vue/src/App.vue:43]: done

**Pull request merged**